packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -57,16 +58,31 @@ function DropdownHeader( {
 	);
 }
 
-function FormField< Item >( {
+function PanelDropdown< Item >( {
```

Comment:
Moved the `Dropdown` code to its own component to better handle the `labelPosition` in the `FormPanelField` component.